app/views/short_url_requests/remove.html.erb

```diff
@@ -0,0 +1,13 @@
+<% breadcrumb :remove_short_url_request, @short_url_request %>
+
+<h1>Delete URL redirect or Short URL <%= @short_url_request.from_path %></h1>
```

Comment:
🤔 seems we use the [locale file](https://github.com/alphagov/short-url-manager/blob/e763146fbd33ebc8f0cb51d9ea4cb43aaf7eb324/config/locales/en.yml) for some things in this app. But not much.

Should content be in the view, or out in the locale file?

Reply:
I think this is consistent with the rest of the app - we only support English right now and I don't anticipate other languages being supported.